src/main/java/aquarium/shop/product/Product.java

```diff
@@ -22,6 +24,19 @@
     @Column(nullable = false)
     private Double price;
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I would remove price from equals method, I believe we shouldn't differentiate 2 products by this field.